### PR TITLE
More tests

### PR DESCRIPTION
**Pull request by prymitive**

### CODE REVIEW

File: alertmanager/remote_test.go [hunk 1]
```diff
@@ -0,0 +1,79 @@
+package alertmanager
```
Comment (jamesog) [inline alertmanager/remote_test.go]: Should this be `alertmanager_test`?

File: transform/slices_test.go [hunk 2]
```diff
@@ -0,0 +1,56 @@
+package transform
```
Comment (Tenzer) [inline transform/slices_test.go]: `package transform_test`?

File: alertmanager/remote_test.go (hunk 1 above)
Comment (prymitive) [inline alertmanager/remote_test.go]: nope, it's testing unexported functions `joinURL` & `getJSONFromURL`

File: transform/slices_test.go (hunk 2 above)
Comment (prymitive) [inline transform/slices_test.go]: `stringInSlice` is not being exported in that package, it it must be `transform`

**Pull request merged**